core/services/vrf/listener_v2.go

```diff
@@ -641,6 +642,7 @@ func (lsn *listenerV2) processRequestsPerSub(
 				ll.Errorw("Error enqueuing fulfillment, requeuing request", "err", err)
 				continue
 			}
+			ll.Infow("Enqueued fulfillment", "ethTxID", ethTX.ID)
```

Comment:
Can we also add this field in the batch fulfillment logs? Should be in `processBatch` in `listener_v2_types.go`

Reply:
done!